cms/templates/js/content-visibility-editor.underscore

```diff
@@ -0,0 +1,33 @@
+<form>
+<h3 class="modal-section-title" id="content_visibility_label"><%- gettext('Student Visibility') %></h3>
```

Comment:
@efischer19 
This label "Student Visibility" reads as "visibility of students" to me - how about "Content Visibility" or "Subsection Visibility" if this label is always at the subsection level?